### PR TITLE
My last pass over the intro (i.e. before the datalad overview)

### PR DESCRIPTION
**Pull request by mih**
Each commit should have a relatively detailed rational re the proposed changes.

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -222,14 +223,15 @@ Such annotation is not sufficient to introduce changes by following the descript
...
-To address this requirement, git-annex established a protocol [@git-annex:special_remotes_protocol] through which external tools can provide custom transport functionality transparently to the git-annex user.
-This allows DataLad and many other projects to facilitate access to an ever-growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inode limits).
+Interoperability with scientific or commercial computing and storage services allows researchers to integrate data management routines into their established workflows with minimal friction.
+Git can already interact with other local or remote repositories via a set of standard (ssh, http) or custom (Git) network transport protocols.
+DataLad implements support for additional services that require custom protocols, such as the Open Science Framework (OSF) [@datalad-osf:zenodo].
+Git-annex readily provides access to a wide range of external data storage resources via a large set of protocols.
+DataLad build on this support and adds, for example, more fine-grained access (e.g.
```
Comment (bpoldrack) [inline paper.md]:
```suggestion
DataLad builds on this support and adds, for example, more fine-grained access (e.g.
```

File: paper.md [hunk 2]
```diff
@@ -157,20 +157,22 @@ DataLad aims to make data management as easy as managing code.
...
+Code, data and computing environments are core components of scientific projects.
+While the collaborative development and use of research software and code is streamlined with established procedures and infrastructures, such as software distributions, distributed version control systems, and social coding portals like GitHub, other components of scientific projects are not as transparently managed or unobstructed accessible.
+Data consumption is complicated by disconnected data portals that require a large variety of different data access and authentication methods.
+(Re)used data are not as precisely identified as it is standard for code in software development, because data versioning is rarely or only coarsely practiced.
+Scientific computation is not reproducible enough, because data provenance, the information of how a digital file came to be, is often incomplete and rarely automatically captured.
+Last but not least, in the absence of standardized data *packages*, there is no uniform way to declare actionable data dependencies and derivative relationships between inputs and outputs of a computation.
```
Comment (bpoldrack) [inline paper.md:172]: Entire Statement of Need reads much better!

File: paper.md [hunk 3]
```diff
@@ -194,24 +196,23 @@ With this simple approach, git-annex enables separate and optimized implementati
...
-Unlike changes to text documents or source code, which are typically done "manually", data manipulations are most often performed by software.
-DataLad exploits this fact and enables automated annotation of changes which result from running an external command.
-DataLad creates a commit message which does not only include a human-readable summary, but also a human- and machine-readable record of the command invocation which introduced the changes.
-This allows for the data "change" to be re-executed to either verify that results reproduce, or to apply such a "change" to a completely different state, irrespective of the quality of the human-made commit message.
+**Re-executable annotation of changes.**
+Digital provenance is crucial for the trustworthiness and reproducibility of a research result, and contributes to the reusability aspect of the FAIR principles [@FAIR2016].
+Git captures provenance with annotations of changes that are typically represented by a patch (an exact difference between two versions that could be applied to another version of the text file), and a commit message (a freeform, human-readable text description of the introduced changes).
```
Comment (bpoldrack) [inline paper.md]: Not really about this PR, but this sentence still is confusing in its structure. How about this?

```suggestion
Git captures provenance with annotations of changes, where changes are typically represented by a patch (an exact difference between two versions that could be applied to another version of the text file), and annotations by a commit message (a freeform, human-readable text description of the introduced changes).
```

**Review by bpoldrack — COMMENTED** [review]
Significant improvement from my POV!

File: paper.md (hunk 3 above)
Comment (mih) [reply paper.md]: The sentence is (without parenthesis):

> Git captures provenance with annotations of changes that are typically represented by a patch, and a commit message.

I think your proposal is more correct, but arguably less attractive ;-)

What about

```suggestion
Git captures provenance by annotating a patch (an exact difference between two versions that could be applied to another version of the text file), with a commit message (a freeform, human-readable text description of the introduced changes).
```

Comment (bpoldrack) [reply paper.md]: Yes, better. :)

**Comment by adswa** [comment]
Cool!

**Comment by yarikoptic** [comment]
I would have preferred to keep dRDM, but since everyone is happy, I am happy.  Thank you @mih !

**Pull request merged**